thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+      ],
+      "enabled": ""
+    },
+    {
+      "title": "Delete Observable",
+      "operation": "delete_observable",
+      "description": "Delete a specific observable from the TheHive platform based on the input parameters you have specified.",
+      "category": "investigation",
+      "annotation": "delete_observable",
+      "output_schema": {
+        "type": "",
+        "message": ""
+      },
+      "parameters": [
+        {
+          "title": "Observable ID",
+          "name": "observableId",
+          "type": "text",
+          "description": "Specify the ID of the observable from which you want to retrieve information from TheHive Platform.",
+          "tooltip": "Specify the ID of the observable from which you want to retrieve information from TheHive Platform.",
+          "visible": true,
+          "editable": true,
+          "required": true
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "title": "Create Task in Case",
+      "operation": "create_task",
+      "description": "Creates a task in the in TheHive based on the input parameters you have specified.",
+      "category": "investigation",
+      "annotation": "create_task",
+      "output_schema": {
+        "_id": "",
+        "_type": "",
+        "_createdBy": "",
+        "_updatedBy": "",
+        "_createdAt": "",
+        "_updatedAt": "",
+        "title": "",
+        "group": "",
+        "description": "",
+        "status": "",
+        "flag": "",
+        "startDate": "",
+        "endDate": "",
+        "assignee": "",
+        "order": "",
+        "dueDate": "",
+        "mandatory": "",
+        "extraData": {}
+      },
+      "parameters": [
+        {
+          "title": "Case ID",
+          "name": "caseId",
+          "type": "text",
+          "description": "Specify the ID of the case in which you want to create task in TheHive Platform",
+          "tooltip": "Specify the ID of the case in which you want to create task in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Title",
+          "name": "title",
+          "type": "text",
+          "description": "Specify the title of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the title of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Description",
+          "name": "description",
+          "type": "text",
+          "description": "Specify the description of the task you want to create in TheHive Platform",
+          "tooltip": "Specify the description of the task you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Status",
+          "name": "status",
+          "type": "text",
```

Comment:
Is there any drop down options for this parameter. If Yes, please provide the options or you can add placeholder.